airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -947,9 +947,7 @@ def execute(self, context) -> None:
                 delegate_to=self.delegate_to,
                 impersonation_chain=self.impersonation_chain,
             )
-            schema_fields = json.loads(
-                gcs_hook.download(gcs_bucket, gcs_object).decode("utf-8")  # type: ignore[attr-defined]
-            )  # type: ignore[attr-defined]
+            schema_fields = json.loads(gcs_hook.download(gcs_bucket, gcs_object))
```

Comment:
The call to `decode` requires `bytes` but the return type of `download` is `Union[str, bytes]`. If we pass `bytes` object to `json.loads`, `decode` is called in `json.loads`, so I think it's better not to care about the returned object's type from `download` in this scope.